configuration/mobile/android-instant.mdx

```diff
@@ -0,0 +1,158 @@
+
+This allows you to:
+
+- Authenticate users quickly, using Rownd
+- Capture user intent with minimal drop-off
+- Seamlessly transition users to the full app after sign-in
+
+---
+
+## Project structure
+
+To build an Instant App, your Android project must use a **modular** structure.
+
+### 🔹 Base module (`:base`)
+
+The base module includes shared code and resources used across your app, including your instant and installable experiences. This includes:
+
+- Shared layouts and assets
+- Common logic/utilities
+- The Rownd SDK (💡Keep it lean!)
+
+### 🔹 Feature modules
+
+Split your app into **feature modules** for each experience:
+
+| Module type | Purpose | Notes |
+|-------------|---------|-------|
+| `:instant-login` | The minimal experience for Rownd login | Must stay under 15MB total size |
+| `:full-app` | The installable full version of your app | Can include everything |
+
+> ⚠️ **Google Play size limit**: Instant app experiences must be ≤15MB total (including the base module). Strip unused assets, fonts, and libraries.
+
+---
+
+## Getting started
+
+### 1. Enable Instant Apps in your project
+
+Follow [Google’s official guide](https://developer.android.com/topic/google-play-instant) to set up:
+
+- Instant app support in your manifest
+- `base`, `feature`, and `instant` module structure
+- Instant app build variants
+
+### 2. Add Rownd to your instant feature
+
+Include the Rownd SDK in your `:base` module (or `:instant-login` if you want more separation). Follow the [Android SDK integration guide](/sdk-reference/mobile/android) to configure Rownd for sign-in.
+
+#### `settings.gradle`
+
+```groovy
+include ':base', ':instant-login', ':full-app'
+```
+
+**base/build.gradle.kts**
+
+```kotlin
+dependencies {
+    implementation("io.rownd:android:<latest-version>")
+    // other shared deps
+}
+```
+
+```kotlin
+class MainApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        Rownd.configure(
+            this,
+            "REPLACE_WITH_YOUR_APP_KEY"
+        )
+    }
+}
+```
+
+#### `instant-login/build.gradle.kts`
+
+```kotlin
+dependencies {
+    implementation(project(":base"))
+    implementation("io.rownd:android:<latest-version>")
+    // other instant app deps
+}
+```
+
+```kotlin
+class InstantLoginActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // Your instant app logic here
+        setContent {
+            InstantLoginScreen(onSignIn = {
+                Rownd.requestSignIn()
+            }) // Your instant app UI
+        }
+    }
+}
+```
+
+### 3. Link to your instant experience
+
+You can generate smart links or deep links to launch your instant app directly from:
+- Your website
+- QR codes
+- Marketing emails and campaigns
+
+Rownd can generate these links automatically, or you can create them manually using App Links.
```

Comment:
Maybe we should link out to these docs on app links for instant apps, since there are a couple small differences: https://developer.android.com/training/app-links/instant-app-links#app_links_overview